python/analysis/interpolation/DualEdgeTriangulation.sip

```diff
@@ -40,8 +40,8 @@ class DualEdgeTriangulation: Triangulation
     virtual double getYMin() const;
     /** Returns the number of points*/
     virtual int getNumberOfPoints() const;
-    /** Sets the behaviour of the triangulation in case of crossing forced lines*/
-    virtual void setForcedCrossBehaviour( Triangulation::forcedCrossBehaviour b );
+    /** Sets the behavior of the triangulation in case of crossing forced lines*/
+    virtual void setForcedCrossBehavior( Triangulation::forcedCrossBehavior b );
```

Comment:
I think here's your error (also TriDecorator.sip) - Triangulation::forcedCrossBehavior is missing the capital for Forced